go/workflow/couler/codegen.go

```diff
@@ -50,6 +50,7 @@ func fillEnvFromSession(envs *map[string]string, session *pb.Session) {
 	fillMapIfValueNotEmpty(*envs, "SQLFLOW_HADOOP_PASS", session.HdfsUser)
 	fillMapIfValueNotEmpty(*envs, "SQLFLOW_submitter", session.Submitter)
 	fillMapIfValueNotEmpty(*envs, "SQLFLOW_WORKFLOW_STEP_LOG_FILE", os.Getenv("SQLFLOW_WORKFLOW_STEP_LOG_FILE"))
+	fillMapIfValueNotEmpty(*envs, "SQLFLOW_WORKFLOW_EXIT_TIME_WAIT", os.Getenv("SQLFLOW_WORKFLOW_EXIT_TIME_WAIT"))
```

Comment:
Maybe we should add a comment to explain this env.

Reply:
Have written here https://github.com/sql-machine-learning/sqlflow/blob/fd336672d6cbc534aeef845c93691e240cdad554/python/couler/couler/steps/sqlflow_step.py#L37